### ISSUE TITLE
Cache folder error for BlenderGIS

### ISSUE THREAD

**Issue by needeco** [domlysz/BlenderGIS#118]
I set up a cache folder destination for the Basemaps in the preferences, yet the error message "Please define a valid cache folder path" still appears.

**Comment by domlysz** [comment]
Can you send me a screenshoot on how the cache folder is set in your addon pref? Is it a relative path? (If so try with an obsolute path)

**Comment by domlysz** [comment]
I make a fix for relative path, gives it a try

cd45ec9

**Comment by needeco** [comment]
Thanks domlysz, its working now.

**Comment by asvpage** [comment]
what should be the cache path ?

**Comment by beastmoister** [comment]
I wanted to use BlenderGIS for the first time. I am on macOS.
I tried to setup the path "/Users/xxxxx/Documents/Blender Cache/GIS Add-On/" for example but the window just closes and 
no path is in the field. Even when trying to paste it into the field manually it immediately vanishes.

Help would be greatly appreciated.

**Comment by stardiviner** [comment]
I still got this error on the latest version Blender 5.0 and BlenderGIS 2.2.10 which downloaded from GitHub Releases page.

I have tried click the folder icon to set cache folder, when I clicked "Accept" in the popup file select window, the cache path not setted. Then I tried to manually fill in path string, still failed to set it.

Here is the error output:

<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/93f8853f-209b-44e7-acdf-ba8689f398d7" />
